__tests__/articles.test.js

```diff
@@ -51,6 +51,31 @@ describe("/api/articles", () => {
             });
           });
       });
+
+      it("should return articles filtered by topic when topic query is provided", () => {
+        return request(app)
+          .get("/api/articles?topic=coding")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body).toHaveProperty("articles");
```

Comment:
we really don't need to be checking the key of articles comes back at this point - the previous tests on this endpoint should cover it but line 62 does the job of needing articles to be the key for the assertion to work.